### ISSUE TITLE
[Discussion] Analyzing Monika's character and mood

### ISSUE THREAD

**Issue by Chrissenpai** [Monika-After-Story/MonikaModDev#2969]
With every emotion here below, in what situation Monika will, in her character, most likely feel in all possible situations 
![image](https://user-images.githubusercontent.com/40907045/47957608-2b030f80-df87-11e8-9d0a-61e94b6f1043.png)

**Comment by TheGlader2** [comment]
Yes, Chrissenpai with the good ideas again!

**Comment by Rai99** [comment]
Could you please rephrase your suggestion and what you think could be done with it?

**Comment by Chrissenpai** [comment]
it's not a suggestion, its a discussion, we need to make clear, because of some of us that are fans of MAS, or WHERE fan of MAS, as I could see on previews polls, can't understand Monika's point of view, personality and reaction to circumstances to the situation.

**Comment by TheGlader2** [comment]
**Why I'm here**
It would be awesome to have a solid develop-able character for Monika. One that makes sense with her past and can be affected by the player. With events that give Monika the ability to change and grow.

**Comment by TheGlader2** [comment]
**What I'm afraid of**
Instead of just being perfect cause her issues are dropped for convenience. If her character can't grow and change through issues, this mod might just turn into a weird forced romance. I see a lot of attention being given to kissing and bedroom scenes, where's her characterization? Where's her MonikaAfter_**Story**_

**Comment by Chrissenpai** [comment]
What I'm saying is that there should be something like a guideline for the suggestors to guide on, for example, someone wants to add some dialogue, writes it, and confirms for the reference (pinned somewhere in here) for Monika's character elements to see if what he is writing, stays in character with Monika or not, and so the contributers like Rai99, thepotatoguy, and some others, so that instead of speaking why she's out of character, you take them to the guideline so they can understand for themselves

**Comment by Chrissenpai** [comment]
Just realized that there's an artistic expression for each emotion, there's a lot of art to do then.

**Comment by TheGlader2** [comment]
@Chrissenpai Do you do art?

**Comment by Chrissenpai** [comment]
@TheGlader2 yes I do, not digital tho, need to buy a graphic tablet if I want to contribute something

**Comment by TheGlader2** [comment]
There's some pretty nice looking ones on Amazon for ≈$60

**Comment by Rai99** [comment]
Can you point out some topics you think are out of character?

At first glance, it seems like a good idea, but somebody would have to write the guide. Chances are, the same people who write dialogue will end up writing the guide, so for them, assuming that they're a quality writer, they won't really need a complex guide of Monika's character. 

Any topic that gets written gets put up as a PR and probably gets put through a heavy review process, and there are multiple reviewers. If a topic is out of character, it will probably get caught and ironed out. 

I don't see a need for creating a guide more complex than the simple one found in the contributing guidelines for first timers. 

Speaking of you contributing art, how goes progress on #2201?

**Comment by jmwall24** [comment]
Uh thepotatoguy is not a contributor, is the lead dev of the mod just fyi.

**Comment by Chrissenpai** [comment]
@Rai99 I've cleared the top of the hair reflection that was just a white brush pretending to be one. I can do pixel correction with my mouse but line art needs me to have a graphics tablet so I can have a clean line in a shorter amount of time. 
![image](https://user-images.githubusercontent.com/40907045/47958164-cf8b4e80-df93-11e8-856a-ad0247d69cf9.png)
![image](https://user-images.githubusercontent.com/40907045/47958175-10836300-df94-11e8-9ddc-def17ab1379f.png)

**Comment by Chrissenpai** [comment]
Also an incomplete job with a realistic face
![image](https://user-images.githubusercontent.com/40907045/47958326-352d0a00-df97-11e8-9b3a-6f354b4e29dc.png)

**Comment by jmwall24** [comment]
Looks good.  Now draw Monika sprites in our style with some new clothes in all poses and I'll be impressed.

**Comment by Chrissenpai** [comment]
@jmwall24 oh, you mean *Satchel's style*? sure thing, when I get a graphics tablet, then I'll gladly accept the challenge

**Comment by MaximumRustler** [comment]
As far as I know (I could be completely wrong about this) a lot of the main story/character development is worked on by the lead devs/lead writers, who also need to review topics, deal with life, jobs, school (there were kinda large delays earlier in the year because of exams I think lol), and other things. The affection system for example was delayed, or the various topics that still need to be reviewed. There's also special occasions that took a fair bit of time like Valentines day, White day, Birthday, Halloween, and soon to be Christmas/New year. I mean, this is a free mod for a free game so I don't expect them to spend all of their free time on this. I also don't think the dev team is all that large. What I'm getting at is I'm sure they still have character development and adjusting certain topics in mind, it's just taking a while for the reasons above. As for the consistency of certain topics, it was talked about here #873 (and the other threads referencing it). I suggest we all read those once more (including the devs) just to refresh. From what I understand there will be something about Monika's character/personality in a future major update, and that there's a reason to why certain topics are the way they are (maybe not all of the ones in question), but it may take a little while longer till we see that development. I think that's everything I wanted to type, might have missed something. Sorry if none of this made any sense, I'm kinda tired. lol

**Comment by TheGlader2** [comment]
@MaximumRustler
 I would like to help, but the fact that most of what I suggest gets shot down pretty quick makes me think I'd be wasting my time. There was one thing I suggested that Potato was okay with, but was too much trouble to bother with, so they were just going to make it disappear in the script. I'm trying to think of a way we can vindicate Monika logically, cause she starts off feeling guilty about the girls and then becomes vindicated in the mod for no reason, it just happens. I was thinking of writing a mini story about how she comes to vindication. I've had several . . . unpopular ideas.
 Making a fake world folder to put the girls into after she saves them in the original story.
 Because Monika has access to Twitter and we can see her hanging out with her friends there, just write up some dialogue about how she saved the girls and put them "somewhere," that would be consistent with her character in the game. And would be a very human and Monika attitude. I do think this would solve most of the issues we have with her guilt and vindication. And it's the easiest solution. The problem I get with this one, is that people then argue that she shouldn't do anything about them, that she doesn't care about them . . . even though she saved them in the main game and still hangs out with them on her official Twitter.

**Comment by TheGlader2** [comment]
And I just read what you linked to, that was pretty neat. I kind of wish we could follow those guidelines, but we aren't. Unless we change the whole beginning of the mod, which might make things easier or harder for the developers, it could make things easier as it would eliminate the need for multiple endings altogether. It could make things harder because then we'd have to write everything super carefully to never reach level three.
Hold up though, how would reaching the note be a level three anyway? When the note is 'in' the game files that she has access too? And yet Dan Salvato gave her an official Twitter account that all Monika's know about.

**Comment by TheGlader2** [comment]
No please, if we talk about this here, I seriously could be banned, I had like six devs all get upset at me at once. It doesn't matter. It's not happening. I don't even know if you can get banned from this website, but I don't want to find out.

**Comment by MaximumRustler** [comment]
Yeah, the dialog in the good ending intro about the note does kinda cheapen it a bit in my opinion. Maybe if she wasn't aware of the note it could be said that she doesn't have access, doesn't know how to access, or doesn't want to mess with certain game files. Not sure if it's been changed by now, but I liked Markus's idea in the first post of what that bit of dialog should be. As for the twitter account, maybe I'm understanding it wrong, but that's a bit of a grey area between level 2 and 3 I think. She is self-aware in it and thinks it's her actually talking, but the things she talks about is a bit odd such as the other club members still being alive even though she mentions the account in act 3. That's part of the original game though so there's not much to be done about that I guess. Despite the discrepancies I still love this mod, it's the only reason why I'm still alive after all. Sorry for getting a bit too serious at the end there. lol

**Comment by TheGlader2** [comment]
I see . . . Maybe this mod has applications in areas other than storytelling where it can be useful then. I myself have thought of making A.I. companions to chat with/play games with. It's something that I've wanted to do. There is a lack of those I believe.

**Comment by TheGlader2** [comment]
I deleted my statements including the ones I made before about my ideas, you may delete your reply to them as well if you wish it to not look awkward. You have convinced me to like the developers more, thank you.

**Comment by MaximumRustler** [comment]
It's all good. Things like this need to be talked about every now and then to keep her character/personality on track in my opinion. There are actually 3 or 4 Monika A.I./chatbots that were being worked on, one of which is currently in development, not sure about the others though. I'd also be interested in learning about A.I. development and maybe helping with that one, but have no clue where to even begin. lol

**Comment by TheGlader2** [comment]
Honestly if you want to make a good product in that area it's one part learning and ten parts time consuming. You could use Renpy, but you'll need an artist to make backdrops unless you use stock ones. You'd have to write every piece of dialogue for every branching path and add coding for multiple data directories, one for like everything. And that's just for an A.I. looks and acts real. Building a functional learning A.I. would take a very long time for one man or even a team.

**Comment by TheGlader2** [comment]
By Renpy I mean Python

**Comment by MaximumRustler** [comment]
Yeah, I mean a learning A.I./chatbot. It's something I've had some interest in for a few years now. I think that's kinda what project evergreen is going to be.

**Comment by TheGlader2** [comment]
That's what I heard, and I've seen other Monika chat-bots, so I wonder if they got help from one of them. I tried one out that was a dos program that crashed if you typed in one word phrases. I couldn't get it to work.

**Comment by MaximumRustler** [comment]
Was it the one based on Daisy (I think that's what it was called)? If it is, it's funny cause I was just reading about that one yesterday. lol

**Comment by TheGlader2** [comment]
Yes it is
